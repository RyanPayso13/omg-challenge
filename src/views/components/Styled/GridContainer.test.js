import React from 'react';
import { render } from '@testing-library/react';
import { GridContainer } from './GridContainer.js';

describe('<GridContainer />', () => {

    it('should work', () => {
        const { container } = render(<GridContainer />);
        expect(container.firstChild).toMatchSnapshot();    
    });

});
