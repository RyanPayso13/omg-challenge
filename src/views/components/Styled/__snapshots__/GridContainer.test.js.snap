// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GridContainer /> should work 1`] = `
.c0 {
  display: grid;
  grid-template-columns: repeat(2,1fr);
  grid-gap: 1rem;
  margin-top: 3rem;
  position: relative;
}

<div
  class="c0"
/>
`;
